app/controllers/CyaCategorisationController.scala

```diff
@@ -70,7 +68,10 @@ class CyaCategorisationController @Inject() (
             }
             .getOrElse {
               dataCleansingService.deleteMongoData(request.userAnswers.id, CategorisationJourney)
-              logErrorsAndContinue("Failed to get categorisation details", continueUrl(recordId))
+              logErrorsAndContinue(
+                "Failed to get categorisation details",
+                routes.CategorisationPreparationController.startCategorisation(recordId)
```

Comment:
Doesn't particularly matter, just wondering if there is a reason to change all these

Reply:
Because there is weird behaviour with the continue url when it is a private val - it misses off part of the url